tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -23,6 +23,8 @@ namespace spirv {
 
 namespace {
 
+const char *glslExtInstSetName() { return "GLSL.std.450"; }
```

Comment:
Hmm, not necessary. Static const global variables won't really hurt I think. But external mutable global variable is definitely evil.